Home form update (big updates)

loganmorganml1 opened the pull request:
I did a lot of formatting with this:
-- The form is now with the hero section, and no need for negative margins 
-- I put the instagram section in with the "other-grid" content to make it flow better and get rid of awkward white space. 
-- deleted negative margins and 1fr rows and columns in code I was working with when not needed 
-- fixed funky subhero pages and deleted unnecessary negative margins (the attributions hero page worked fine but the others got wonky, so I used the code for the header in the attribution on those pages and updated the content for the page)

There is some weird stuff, like to center the button for mobile I had to make it flex and auto margins, but it looked weird for other screen sizes since the type in the button would align left no matter what I did, so for those the display is inline-block. At this point I just got the buttons to a spot that was easy to put them in and looked okay. 

Please review -- everything is working and looks okay to me but I worked through a lot of code so don't want to have accidentally missed something important.

lala-laura commented:
@loganmorganml1 I'll be wrapping up work in about an hour and will review

lala-laura commented:
@loganmorganml1 it looks like the last merge with the link fix and insta-clean up aren't here. There might be conflicts with current/incoming changes. I don't mind doing making the merge, but let me know if you'd rather.

loganmorganml1 commented:
> @loganmorganml1 it looks like the last merge with the link fix and insta-clean up aren't here. There might be conflicts with current/incoming changes. I don't mind doing making the merge, but let me know if you'd rather.

I just marked it ready, so let me just fix the conflicts first.

Okay, I see the web resolve conflicts button now. I'll check back.

loganmorganml1 commented:
> > @loganmorganml1 it looks like the last merge with the link fix and insta-clean up aren't here. There might be conflicts with current/incoming changes. I don't mind doing making the merge, but let me know if you'd rather.
> 
> I just marked it ready, so let me just fix the conflicts first.
> 
> Okay, I see the web resolve conflicts button now. I'll check back.

@lala-laura Alright I've merged and pulled it up, and fixed any conflicts that I saw. All links seem to be working. I also cleaned up the instagram CSS by putting it all close together, since there was an issue with the layout at first (turns out it was the links in the HTML) and I couldn't figure out what it was, so now insta CSS for mobile, tablet, and desktop are right below each other. Let me know if I've missed any periods, I think I got them all correct, but an extra set of eyes would be appreciated.

lala-laura commented on about/index.html:
```diff
@@ -14,43 +14,43 @@
+       <!-- Header -->
+       <section>
+           <div class="nav-container">
+               <a href="/index.html"><img src="/assets/img/logos/logo-icon/logo-icon-white.svg" alt="Who Travels Logo" class="logo"></a>
```
about nav - index and logo missing link fix update

loganmorganml1 replied:
@lala-laura just updated, also fixed some links in destinations and contact

lala-laura commented:
Sorry, I'm not trying to take forever. I tried to load the site in my browser (not using live server) and the home page looks great but the other pages aren't loading. When I opened 'book' from it's own index file css was missing

loganmorganml1 commented:
> Sorry, I'm not trying to take forever. I tried to load the site in my browser (not using live server) and the home page looks great but the other pages aren't loading. When I opened 'book' from it's own index file css was missing

from the nav they aren't working? there's only "." on the main index nav, maybe that's why. I can update that. I'll go into book and double check for any other link issues, since idk why it would effect css without it.

Feel free to open my code in VS code as well if it will help you see where I may be messing up.

loganmorganml1 commented:
> Sorry, I'm not trying to take forever. I tried to load the site in my browser (not using live server) and the home page looks great but the other pages aren't loading. When I opened 'book' from it's own index file css was missing

I'll be gone for about an hour and a half but be back to fix anything else. I haven't updated the nav yet for home, but added double periods that were missing in booking. And no worries for taking your time, I'd rather get this right than have it merged right away! It seems to be just linking issues right now, so it shouldn't take too long. Just got to make sure we find them all

@loganmorganml1 Okay, sounds good. I'm going to keep my computer up but I also need to step away for a just a bit. But not too long. We'll get this!

techeduk8r commented on about/index.html:
```diff
@@ -14,43 +14,43 @@
+       <div class="hero-subpage">
+           <section class="hero">
+               <div class="hero-img-box">
+                   <img class="hero-img" src="../assets/img/heros/hero-about.jpg" alt="Travelers">
```
The alt should be set back to Galaxy.

loganmorganml1 replied:
updated

techeduk8r commented on book/index.html:
```diff
@@ -35,19 +36,20 @@
+        <div class="hero-subpage">
+            <section class="hero">
+                <div class="hero-img-box">
+                    <img class="hero-img" src="../assets/img/heros/booking-hero.jpg" alt="Travelers">
```
Again, the alt tag should be changed.

loganmorganml1 replied:
updated

techeduk8r commented on contact/index.html:
```diff
@@ -35,20 +35,19 @@
+        <div class="hero-subpage">
+            <section class="hero">
+                <div class="hero-img-box">
+                    <img class="hero-img" src="../assets/img/heros/hero-contact.jpg" alt="Travelers">
```
Same w/alt tag. Even though this is a hero image, it's an image on the page and needs to reflect its content. That's an aspect of accessibility he may look at.

loganmorganml1 replied:
updated

techeduk8r commented on destinations/index.html:
```diff
@@ -13,43 +13,43 @@
+       <div class="hero-subpage">
+           <section class="hero">
+               <div class="hero-img-box">
+                   <img class="hero-img" src="../assets/img/heros/hero-destinations.jpg" alt="Travelers">
```
alt tag

techeduk8r commented on assets/css/style.css:
```diff
@@ -1609,6 +1581,63 @@ margin-top: 5px;
   padding-left:5px;
 
 }
+@media (min-width: 768px) {
```
At mobile size, the IG images hang over the edge a bit. Maybe it should be two columns or move the title above the images.

loganmorganml1 replied:
I figure you meant ipad -- I just updated the grid to a new look so that the text is on top. I also centered it because it felt a bit off. Since I centered it for ipad I have it for desktop to. Let me know what you think of these changes, I can always revert back if we don't like the look

techeduk8r reviewed (changes requested):
See inline comments.

techeduk8r commented:
Once these updates are good to go, I can fix the accessibility and HTML validation issues. I started but realized some of this would affect both.

lala-laura commented on index.html:
```diff
@@ -46,37 +46,37 @@ <h1 class="hero-h1">Travel</h1>
                     <h2 class="hero-h2"> Who Travels?</h2>
                     <p class="hero-body">It could be you! We're the one stop shop for all time and space travel needs.</p>
                 </span>
+                 <div class="entire-form-content">
```
I'm seeing it in the header on both

loganmorganml1 replied:
Does this happen for all the pages? This is what it was like before I put the code in with the hero -- when I integrate it with the hero, it's in the right spot and eliminates the need for negative margins (also interesting the button is all the way to to the right -- that was done with janky code I took out, which is why it now rests below the form fill in items)

techeduk8r replied:
This only occurred on the homepage. If it looks fine on both of your devices, then it might just be something weird with mine.

loganmorganml1 commented on index.html:
```diff
@@ -46,37 +46,37 @@ <h1 class="hero-h1">Travel</h1>
+                                 </div>
+                             </div>
+                         </form>
+                          <div class="search-form-button">
```
It's because its working on a two row single column grid. I did this for simplicity sake to make sure the code was clean, since I was having trouble trying to work with the button and form side by side -- I couldn't get the widths of the columns with percentages or fr. So now the button and form are together in a div. 
<img width="1101" alt="Screen Shot 2021-12-14 at 6 56 16 PM" src="https://user-images.githubusercontent.com/93295768/146103102-6915c1ca-b818-4c4e-b90b-0da8d1362f6c.png">

techeduk8r replied:
Cool. Just wanted to make sure this was intended. Might it be possible to use a text-align on the button's class to align it to the right?  It makes for a more logical flow.

loganmorganml1 replied:
Let me try, because I agree. It would look better to the right.

loganmorganml1 replied:
Perfect, it worked! I see what I did wrong now -- I was calling the class with the button, instead of just the class. Button should be right aligned now.

loganmorganml1 commented on index.html:
```diff
@@ -153,38 +153,37 @@ <h2 class="pop-destinations-heading">Popular Destinations</h2>
-                <h3 class="insta-name">Instagram</h3>
-                <h3 class="insta-hashtag">#WhoTravels</h3><br>
-            </div>
+                <div class="insta-item">
```
just updated!

techeduk8r reviewed (commented):
If the homepage looks good to both of you, I'm fine with it. See my inline comment. 

For the homepage search button, I made one suggestion. Take it or leave it.

loganmorganml1 commented:
@lala-laura @techeduk8r Made necessary updates! Let me know if there is anything else.

loganmorganml1 commented:
Ah just saw this -- let me take the code from that merge first and put it in here.
________________________________
From: lala-laura ***@***.***>
Sent: Tuesday, December 14, 2021 3:55 PM
To: techeduk8r/who-travels ***@***.***>
Cc: Logan, Morgan ***@***.***>; Mention ***@***.***>
Subject: Re: [techeduk8r/who-travels] Home form update (big updates) (PR #54)


@loganmorganml1<https://github.com/loganmorganml1> it looks like the last merge with the link fix and insta-clean up aren't here. There might be conflicts with current/incoming changes. I don't mind doing making the merge, but let me know if you'd rather.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/techeduk8r/who-travels/pull/54#issuecomment-994075738>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWHZJGFEGG4BZH4TICJHLALUQ64NPANCNFSM5J752WJA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

loganmorganml1 commented:
Oh sorry I was replying to Briana's message about the insta clean up! Let me know if that is still an issue -- I did go in and fix the links when I noticed, and copied the CSS from the main.
________________________________
From: Logan, Morgan ***@***.***>
Sent: Tuesday, December 14, 2021 7:38 PM
To: techeduk8r/who-travels ***@***.***>; techeduk8r/who-travels ***@***.***>
Cc: Mention ***@***.***>
Subject: Re: [techeduk8r/who-travels] Home form update (big updates) (PR #54)

Ah just saw this -- let me take the code from that merge first and put it in here.
________________________________
From: lala-laura ***@***.***>
Sent: Tuesday, December 14, 2021 3:55 PM
To: techeduk8r/who-travels ***@***.***>
Cc: Logan, Morgan ***@***.***>; Mention ***@***.***>
Subject: Re: [techeduk8r/who-travels] Home form update (big updates) (PR #54)


@loganmorganml1<https://github.com/loganmorganml1> it looks like the last merge with the link fix and insta-clean up aren't here. There might be conflicts with current/incoming changes. I don't mind doing making the merge, but let me know if you'd rather.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/techeduk8r/who-travels/pull/54#issuecomment-994075738>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWHZJGFEGG4BZH4TICJHLALUQ64NPANCNFSM5J752WJA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

lala-laura commented:
I think we should pull this branch before it becomes a tree.

lala-laura commented:
@loganmorganml1 @techeduk8r I don't see any glaring problems from my end? 

Briana is there any outstanding issue with this branch still that you noticed?

loganmorganml1 commented:
> I think we should pull this branch before it becomes a tree.
I laughed thinking this was a pun, and then thought maybe I'm dumb and there is something about trees on github lol.

techeduk8r reviewed (approved):
Looks okay to me.

techeduk8r commented:
Tree averted!
